src/containers/pages/Home/Home.tsx

```diff
@@ -33,7 +44,19 @@ class Home extends React.Component<{}, {}> {
           </Col>
         </Row>
         <Row className="intervention-box">
-          {ENABLE_IRS && (
+          {ENABLE_HOME_MANAGE_PLANS_LINK && (
```

Comment:
how best can we harmonize the appearance of this link to that in `Header` component,

Reply:
My thinking was to decouple the dependency so that if we ever get similar request we only change the envs.

Reply:
hmmm,..